app/controllers/application_controller.rb

```diff
@@ -28,7 +28,6 @@ def prevent_caching_via_headers
   def csp_headers
     response.headers['Content-Security-Policy-Report-Only'] =
       "default-src 'self'; " \
-      "script-src 'self' www.google-analytics.com 'sha256-1kYydMhZjhS1eCkHYjBthAOfULylJjbss3YE6S2CGLc=' 'unsafe-eval'; " \
```

Comment:
I think we can just ditch the `www.google-analytics.com` part of this and keep the `script-src` stuff around it. Basically what this is doing is it's throwing a flag if any javascript loads that doesn't have the `sha` hash of `1kYydMhZjhS1eCkHYjBthAOfULylJjbss3YE6S2CGLc`. 

You can check out some more info on CSPs here: https://content-security-policy.com/